ex1-shai/bootstrap.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
See the comment about the shebang in Liroy's PR

Reply:
Change to - #!/usr/bin/env bash?

Reply:
yes. Doing it in the following PR.